### PR TITLE
small gui and language fixes

### PR DESCRIPTION
**Pull request by thohemp**

### CODE REVIEW

**Comment by aitorp6** [comment]
Can you explain to us your changes?

**Comment by thohemp** [comment]
In some cases the translation didn't work, because the phrases in the gui files and language files didn't match. In this case those particular text phrases remained english. I unified those phrases to activated the translation and show the text in the local language.

In the current wallet gui some button text of the toolbar buttons are cut at the end, so I increased the toolbar width a little bit.

File: src/qt/locale/bitcoin_fr.ts [hunk 1]
```diff
@@ -221,7 +221,7 @@
     </message>
     <message>
         <source>Warning: The Caps Lock key is on!</source>
-        <translation>Avertissement : la touche Verr. Maj. est activée !</translation>
```
Comment (aitorp6) [inline src/qt/locale/bitcoin_fr.ts]: what are these changes?

File: src/qt/addressbookpage.cpp [hunk 2]
```diff
@@ -21,6 +21,8 @@
 #include <QMessageBox>
 #include <QSortFilterProxyModel>
 
+
```
Comment (aitorp6) [inline src/qt/addressbookpage.cpp]: why are here these spaces?

Comment (thohemp) [reply src/qt/addressbookpage.cpp]: Nonsenee, might be an issue of search & replace. I will close this and reopen later.

**Comment by aitorp6** [comment]
Thank you for your contribution.

One question, which is your name on our forum?

**Comment by thohemp** [comment]
Obszoenling

**Pull request merged**